recipe/install_pkg.bat

```diff
@@ -20,6 +20,14 @@ for /L %%I in (18,1,%PKG_VERSION:~0,2%) do (
     if %ERRORLEVEL% neq 0 exit 1
 )
 
+:: this package creates a copy of 'libomp.dll' and installs it as 'libiomp5md.dll' to
+:: replace Intel's openmp library. The issue with copying is that if a process loads
+:: both 'libimp5md.dll' and 'libomp.dll', one of them will complain that another OpenMP
+:: runtime is being loaded even though they are literally the same.
+:: To avoid this, let's make 'libiomp5md.dll' a DLL that forwards to 'libomp.dll'
```

Comment:
cc @h-vetinari

Reply:
Ah, thanks. I hadn't looked into the commit `no need to set arch` because the title implied it would just remove a CLI argument